beamline.rst

```diff
@@ -65,12 +61,6 @@ generally with the beamline.
 
         These must all have a allowed_values or enum
 
-        # NB we should standardise the vocabularies as well as the names.
```

Comment:
Lines 68-73 raise a point that is not decided and needs to be considered. Where should that be reflected?

Reply:
We should maybe make it an issue so that it can be discussed in detail, I just created #22 please feel free to complete it.